web/tests/choice-params-with-default.spec.ts

```diff
@@ -4,7 +4,7 @@ import { Workspace } from './lynxkite';
 
 let workspace: Workspace;
 test.beforeAll(async ({ browser }) => {
-  workspace = await Workspace.empty(browser);
+  workspace = await Workspace.empty(await browser.newPage());
```

Comment:
Sorry about all this! I noticed that quite often we have a single test in a file. In this case won't re-use a global object, so we don't need this `newPage()` call. I think passing in a `Page` is a better interface. It allows resetting the test environment inside a test too.

Reply:
So do you suggest passing in a Page in files with a single test and syntax like this in files with multiple tests?

Reply:
> So do you suggest passing in a Page in files with a single test and syntax like this in files with multiple tests?

Yes.